playbooks/setup_forklift.yml

```diff
@@ -41,3 +42,15 @@
         # it can be removed when we use ansible-lint 4.3.0 as that contains a fix
         content: "{{ forklift_config | to_json | from_json | to_nice_yaml }}"
         dest: "{{ forklift_dest }}/vagrant/settings.yaml"
+
+    - name: 'Install Forklift collection dependencies'
+      command:
+        cmd: ansible-galaxy collection install -r requirements.yml
+        chdir: "{{ forklift_dest }}"
+      when: forklift_install_from_galaxy
+
+    - name: 'Install Forklift pulp_installer role dependencies'
+      command:
+        cmd: ansible-galaxy role install -r playbooks/galaxy_collections/ansible_collections/pulp/pulp_installer/requirements.yml
+        chdir: "{{ forklift_dest }}"
+      when: forklift_install_from_galaxy
```

Comment:
We can change this later, but I forsee a future where we will want a specific install from galaxy option for just Pulp assets.

Reply:
I'd prefer to change that when required.